simulate.py

```diff
@@ -82,10 +82,10 @@ def simulate(args):
                 voltage_level=gc.voltage_level,
                 interval=s.interval,
                 timestamps_list=timeseries.get("time"),
-                power_grid_supply_list=timeseries.get("grid power [kW]"),
+                power_grid_supply_list=timeseries.get("grid supply [kW]"),
                 price_list=timeseries.get("price [EUR/kWh]"),
-                power_fix_load_list=timeseries.get("ext.load [kW]"),
-                charging_signal_list=timeseries.get("window"),
+                power_fix_load_list=timeseries.get("local generation [kW]"),
```

Comment:
same as above: Shouldn't this be "fixed load"?

Reply:
yes. done.